### PR TITLE
Announcing new strategy

### PR DESCRIPTION
**Pull request by elshafeh**
The end of an era? :)

### CODE REVIEW

**Comment by robertoostenveld** [comment]
I will make some updates to this, but am busy at the moment. Please don't merge yet.

**Comment by robertoostenveld** [comment]
@schoffelen and @Spaak what do you think of this?

- the tweet-on-every-commit has just been disabled in the fieldtrip/fieldtrip webhook. 
- the tweet-on-newsitem now works (confirmed with an old release PR, but I deleted the tweet again)
- the fieldtrip [homepage](http://www.fieldtriptoolbox.org) has been updated with a more central place for the latest (important) release and links to twitter and GitHub

File: _posts/2020-09-17-strategy-change.md
Comment (schoffelen) [inline _posts/2020-09-17-strategy-change.md:9]: In addition to 'systematically', I'd say that twitter is going to be used more to-the-point

File: development/guideline/communication.md
Comment (schoffelen) [inline development/guideline/communication.md:11]: perhaps be really explicit that it's not users versus support, but also to promote interactions between users and users...

File: development/guideline/communication.md
Comment (schoffelen) [inline development/guideline/communication.md:14]: Perhaps move this to a separate item, because above it suggests that this is a 'used' channel, which it actually is not.

File: development/guideline/communication.md
Comment (schoffelen) [inline development/guideline/communication.md:22]: typo: 'send' should be 'sent' (3 times)

File: development/guideline/communication.md
Comment (schoffelen) [inline development/guideline/communication.md:24]: here send is correctly spelled

**Review by schoffelen — CHANGES_REQUESTED** [review]

**Comment by schoffelen** [comment]
I think the documentation (with a few typos) is overall good. I have left a few small comments

**Comment by elshafeh** [comment]
And as far as I'm concerned, I'd also like to hear from @schoffelen and @Spaak about whether the twitter page should exist as an automated entity (status quo) or a hybrid i.e. allowing the account to engage with the community.

**Comment by schoffelen** [comment]
what's the purpose of the 'twitter page'?

**Comment by elshafeh** [comment]
That's basically my question. Is the purpose just to announce fieldtrip things? If so, do we want this to be only in a "new version out" or "new workshop" announced, or with some leeway which could involve something between retweeting somebody else mentioning fieldtrip or posting a photo of some post-corona social moment? And more importantly (to me), do we wanna stop there or engage in ongoing social/political/scientific issues e.g. diversity, open science or whatever?

**Comment by Spaak** [comment]
Hey, I'm away today but I'll have a look tomorrow.

File: _posts/2020-09-17-strategy-change.md
Comment (robertoostenveld) [reply _posts/2020-09-17-strategy-change.md]: resolved in https://github.com/fieldtrip/website/pull/288/commits/8deb759fea53e521835c3552d96f00bc79b92407

File: development/guideline/communication.md
Comment (robertoostenveld) [reply development/guideline/communication.md:14]: We can also just leave FB out here since there is indeed no strategy for it.

File: development/guideline/communication.md
Comment (robertoostenveld) [reply development/guideline/communication.md:23]: one out of four, so I am performing below chance level.

**Comment by robertoostenveld** [comment]
what IS the "twitter page"? Do you mean a markdown document in the   https://github.com/fieldtrip/website/tree/master/_posts directory?

Am I right that your question is: should we allow tweets only to be made (1) through the currently implemented mechanism, or should we allow for tweets to be made in (2) the twitter application by anyone who knows the password?

The advantage that I see for (1) is that it allows for review, contributions/corrections from others, and makes people accountable. A shared twitter password as in (2) means that we don't know who has tweeted (although some AI easily resolves that: if it contains typo's it was probably Robert, if it contains exclamation marks and emojis it was probably Hesham 😉 ).
 
Another argument I see for (1) is that I am not aware of things we would want to send out on Twitter that are _not_  suitable for the news section on the homepage, or vice versa; using (1) increases the chances of doing consistent communication to the (partially non-overlapping) groups of people that follow either channel.

**Comment by robertoostenveld** [comment]
> That's basically my question. Is the purpose just to announce fieldtrip things? If so, do we want this to be only in a "new version out" or "new workshop" announced, or with some leeway which could involve something between retweeting somebody else mentioning fieldtrip or posting a photo of some post-corona social moment? And more importantly (to me), do we wanna stop there or engage in ongoing social/political/scientific issues e.g. diversity, open science or whatever?

I think that newsworthy is what we (as the loosely defined FT team) do ourselves. Sharing open source code, contributing knowledge and organizing events fall under that. Liking and/or retweeting is an interesting way to engage with others on Twitter and to amplify our presence. However, the FT project is a MATLAB toolbox for data analysis, not a news broker, hence I am hesitant. So far retweeting has not happened much AFAIK, there are also not that  many [tweets mentioning FT](https://twitter.com/search?q=(to%3Afieldtriptoolbx)&src=typed_query&f=live).

For communicating social aspects we had (once-upon-a-time) FB, which has been (more-or-less) abandoned. Sharing photos became more problematic with the GDPR: I received and honored multiple take-down requests over recent years. As an organization (FT/Donders/RU) we have responsibilities that go beyond those of an individual FT or twitter user. But   
 I would welcome a strategy for this. Who is going to organize the social events, who gets invited, how to deal with consent, who disseminates, etc.

As a project we should _act_ in social/political/scientific issues, like in sharing our code as open-source, and like here https://github.com/fieldtrip/fieldtrip/pull/1468 in relation to BLM. As individuals, we should also act and can also talk/write about these issues. But as academics we cannot hide behind an anonymous or shared account, the authorship and accountability should be clear; I would not want anyone in the (open and voluntarily) FT team to be blamed for the opinion of someone else.

File: development/guideline/communication.md
Comment (Spaak) [inline development/guideline/communication.md:23]:
```suggestion

To add a news item to the website, please file a pull request adding a new file to the [_posts folder on the website](https://github.com/fieldtrip/website/tree/master/_posts). See example files there for how these (markdown) files should be structured. The latest files flagged as `categories: [news]` will, when merged, appear on the homepage.

Whenever a new release is made following automatic code [testing](http://www.fieldtriptoolbox.org/development/testing/), a news item is automatically made that is flagged as `categories: [release]` in the header, and sent as pull request to the [website repository](https://github.com/fieldtrip/website/pulls). This PR has to be reviewed and summarized for human readability by one of the website editors. If merged, the "latest release" section on the homepage will automatically be updated to reflect this.

All news items (both "manually created" and those corresponding to new releases) can contain a `tweet` element in the header. This can contain up to 280 characters for the tweet that is automatically sent once the news item is merged with the website repository. At that moment the news item appears on the website, and a tweet is sent out.

If you only want to send a tweet, you can make a news item without either `news` or `release` as category in the page header (or without any `categories` line at all), but with a `tweet` in the header. Idem for a news item for the website without a tweet: just leave out the `tweet` in the page header and only add the relevant category.
```

File: _posts/2020-09-17-strategy-change.md
Comment (Spaak) [inline _posts/2020-09-17-strategy-change.md:3]:
```suggestion
categories: [news]
```

File: development/guideline/communication.md
Comment (Spaak) [inline development/guideline/communication.md:15]:
```suggestion
With each of these we reach out to a specific group FieldTrip stakeholders but - alhough they are partially overlapping - there is none through which we reach everyone.
```

**Review by Spaak — CHANGES_REQUESTED** [review]
I made the distinction between categories `release` and `news` more explicit, as before my suggested changes this was unclear (I had to look up the homepage markdown template to understand exactly what was what).

**Comment by Spaak** [comment]
@schoffelen:
> what's the purpose of the 'twitter page'?

There is no 'twitter page'.
image

We have our twitter account (is that what you meant?), but on the website there are post items in the `_posts` directory; if one is added (i.e. merged) there that contains a `tweet` header then a tweet will be automatically sent out.

@robertoostenveld @elshafeh 
> Am I right that your question is: should we allow tweets only to be made (1) through the currently implemented mechanism, or should we allow for tweets to be made in (2) the twitter application by anyone who knows the password?

I think that is indeed the question. Arguments in favour of (2) can be given in response to:

> I am not aware of things we would want to send out on Twitter that are not suitable for the news section on the homepage, or vice versa.

My feeling is (which I suspect is shared by the vast majority of Twitter users) that the threshold for something "tweetable" is lower than the threshold for posting a news item on the website. For example, perhaps MNE-python adds some nifty new functionality, they tweet about it, we can reply or quote-tweet: "This looks great from our friends at @MNEpython!". Or one of us tweets from our own account about a new publication featuring interesting (novel) use of FT code, then we can retweet these to give exposure to our contributors. Or, after our twitter account is back into "active communication" mode (i.e., not muted by most), other toolboxes' accounts might tag us, ask questions (not of the support-seeking type), etc., and we might want to reply. (Or, finally, we might simply "like" stuff on twitter to show engagement with the community.)

These different use cases are very easy to handle inside the Twitter web application itself, and this ease of use is precisely why tweeting is generally a fairly low-threshold business. At least some of this is also achievable through the new PR-merge-tweet functionality, but it would take minutes, rather than seconds, to do, plus there is the wait for approval. The *lively* nature of typical twitter engagement will be considerably reduced through this route.

**However** I think that @robertoostenveld raises good points about accountability and reviewability. The new option of adding a post that *only* results in a tweet, without a news item, seems like a reasonable compromise to me. Depending on the details of the twitter library that handles these PR tweets, it should be possible to tweet in-reply-to something, or even send out likes, through this PR mechanism. Something to be looked into (both technically and whether it's desirable) after we settle on this in general, I think.

All in all, LGTM (after merging my requested changes).

**Pull request merged**
